src/Microsoft.TemplateEngine.Cli/CommandParsing/CommandParserSupport.cs

```diff
@@ -162,6 +162,8 @@ private static Option[] NewCommandHiddenArgs
                     Create.Option("-all|--show-all", string.Empty, Accept.NoArguments()),
                     Create.Option("--allow-scripts", string.Empty, Accept.ZeroOrOneArgument()),
                     Create.Option("--baseline", string.Empty, Accept.ExactlyOneArgument()),
+                    Create.Option("-up|--update", string.Empty, Accept.NoArguments()),
+                    Create.Option("-un|--update-no-prompt", string.Empty, Accept.NoArguments()),
```

Comment:
There had been a push to stick with single character switches in the case of the single dash usage. We may just want to give long forms for now (or pick a single character short form).

Reply:
Removed the short versions.